### PR TITLE
Smarty component

### PR DESCRIPTION
**Pull request by z0mbieprocess**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#16178

## Checklist:

- [X] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [X] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/fan.smarty.markdown [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+title: "Salda Smarty 2X/3X/4X P/V Ventilation"
+description: "Instructions on how to integrate Salda Smarty 2X/3X/4X P/V ventilation systems into Home Assistant."
+date: 2018-08-26 20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: salda.png
+ha_category: Hub
+ha_release: "0.77"
```
Comment (DubhAd) [inline source/_components/fan.smarty.markdown]: No longer 0.77

File: source/_components/sensor.smarty.markdown [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+title: "Salda Smarty 2X/3X/4X P/V Ventilation"
+description: "Instructions on how to integrate Salda Smarty 2X/3X/4X P/V ventilation systems into Home Assistant."
+date: 2018-08-26 20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: salda.png
+ha_category: Hub
+ha_release: "0.77"
```
Comment (DubhAd) [inline source/_components/sensor.smarty.markdown]: No longer 0.77

File: source/_components/smarty.markdown [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+title: "Salda Smarty 2X/3X/4X P/V Ventilation"
+description: "Instructions on how to integrate Salda Smarty 2X/3X/4X P/V ventilation systems into Home Assistant."
+date: 2018-08-26 20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: salda.png
+ha_category: Hub
+ha_release: "0.77"
```
Comment (DubhAd) [inline source/_components/smarty.markdown]: No longer 0.77

File: source/_components/smarty.markdown [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+```yaml
+smarty:
+  name: Smarty 3X
+  slave: 1
+```
+
+Configuration variables:
+
+- **name** (*Optional*): The name of this device as you want to see it in Home Assistant.
```
Comment (DubhAd) [inline source/_components/smarty.markdown]: Please use a configuration block, as [in the doc standards](https://developers.home-assistant.io/docs/en/documentation_standards.html#component-and-platform-pages).

**Review by DubhAd — CHANGES_REQUESTED** [review]

**Review by DubhAd — DISMISSED** [review]

File: source/_components/smarty.markdown [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+  slave: 1
+```
+
+Configuration variables:
+
+{% configuration %}
+name:
+  description: The name of this device as you want to see it in Home Assistant.
+  required: false
+  default: smarty
```
Comment (tjorim) [inline source/_components/smarty.markdown]: Default is Smarty with a capital letter according to the code.

Comment (tjorim) [inline source/_components/smarty.markdown]: Default is Smarty with a capital letter according to the code.

File: source/_components/smarty.markdown [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+To set it up, add the following information to your `configuration.yaml` file:
+
+```yaml
+smarty:
+  name: Smarty 3X
+  slave: 1
+```
+
+Configuration variables:
```
Comment (tjorim) [inline source/_components/smarty.markdown]: This line is not needed, the `{% configuration %}` will put it in here.
(when removing this remember to delete one empty line as well)

**Review by tjorim — CHANGES_REQUESTED** [review]
@z0mbieprocess, thanks for your first documentation PR!
I requested some small changes, good luck.

⚠️ The `ha_release` will need an update before merging this one in.

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by frenck** [comment]
Re-opening PR, since the parent PR is still open.

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Closing this PR, since the parent PR has been closed.

**Pull request closed without merging**